src/main/java/utils/AttachmentUtils.java

```diff
@@ -0,0 +1,11 @@
+package utils;
```

Comment:
Please, create a new package - com.epam.tc.hw4 - for all classes for homework 4. Moreover, you have to copy all you page objects there...